src/blocksparse/blocksparsetensor.jl

```diff
@@ -916,6 +945,14 @@ function contract(
   #end
 end
 
+# <fermions>
+function compute_alpha(
+  ElR, labelsR, blockR, indsR, labelsT1, blockT1, indsT1, labelsT2, blockT2, indsT2
```

Comment:
Maybe could have the signature `compute_alpha(ElR, args...)`.

Reply:
For this one, when it's a prototype of the function to be overloaded by an external library and not just a pass-through (like a duplicate interface to some 'core' function), I was thinking it provides both a kind of documentation of what arguments are expected and enforces consistency, since `compute_alpha` gets called from multiple places and we woudn't want those to get out of sync.